en_US/admin/api.md

```diff
@@ -203,25 +231,63 @@ axios
 :::
 ::::
 
-## HTTP Headers
+### Authentication Using Bearer Token
 
-Unless otherwise specified, most API requests require the `Accept` header to be set to `application/json`, and then the response will be returned in JSON format.
+Alternatively, you can use bearer tokens for programmatic and secure authentication. Here's how to obtain and use a bearer token.
 
-## HTTP Response Status Code
+#### Obtain a Bearer Token
 
-EMQX follows the [HTTP Response Status Code](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status) standard, and the possible status codes are as follows:
+To obtain a bearer token programmatically for the EMQX REST API, follow these steps:
 
-| Codes | Description                                                  |
-| ----- | ------------------------------------------------------------ |
-| 200   | Request successfully, and the returned JSON data will provide more details |
-| 201   | Created successfully, and the new object will be returned in the Body |
-| 204   | Request successfully. Usually used for delete and update operations, and the returned Body will be empty |
-| 400   | Bad Request. Usually request body or parameter error         |
-| 401   | Unauthorized. API key expires or does not exist.             |
-| 403   | Forbidden. Check if the object is in use or has dependency constraints. |
-| 404   | Not Found. You can refer to the `message` field in the Body to check the reason |
-| 409   | Conflict. The object already exists or the number limit is exceeded |
-| 500   | Internal Server Error. Check the reason in the Body and logs |
+##### Use cURL
```

Comment:
To get the Bearer token, just need to send an HTTP request to call the login API. You only need to clearly describe this API endpoint, or link directly to the login API documentation. Examples with curl or shell commands are unnecessary, as these are just two of many tools for calling an API. Other tools, like Postman, or even programming methods, can also be used to make the API call. So, the main focus here is to describe the login API interface clearly, not to show how to use a specific tool to call it.